mas-connecta/src/main/java/com/ca/mas/connecta/client/MASConnectaManager.java

```diff
@@ -68,6 +68,8 @@ private MqttConnecta getMqttConnecta() {
         if ((connectOptions == null) || (connectOptions.getServerURIs() == null)) {
             return new GatewayMqttConnecta();
         } else {
+            if (clientId == null)
+                throw new NullPointerException("clientId");
```

Comment:
I need null check here because not relevant to ticket logs

Reply:
Made the required changes as per the comments.